src/types.rs

```diff
@@ -830,27 +1040,37 @@ impl Term {
             _ => None,
         }
     }
-    /// Compute the [alpha equivalence] for two terms.
+    /// Compute the [alpha equivalence] for two `Term`s.
     ///
     /// [alpha equivalence]: https://en.wikipedia.org/wiki/Lambda_calculus#Alpha_equivalence
     /// # Examples
     ///
     /// ```
-    /// # use term_rewriting::{Signature, parse_term,Term};
+    /// # use term_rewriting::{Signature, parse_term, Term, Variable};
+    /// # use std::collections::{HashMap, HashSet};
     /// let mut sig = Signature::default();
-    /// let a = sig.new_op(2, Some(".".to_string()));
     /// let s = sig.new_op(0, Some("S".to_string()));
-    /// let k = sig.new_op(0, Some("K".to_string()));
     ///
     /// let t = parse_term(&mut sig, "S K y_ z_").expect("parse of S K y_ z_");
-    /// let t2 = parse_term(&mut sig, "S K y_ z_").expect("parse of S K y_ z_");
+    /// let t2 = parse_term(&mut sig, "S K a_ b_").expect("parse of S K a_ b_");
     /// let t3 = parse_term(&mut sig, "S K y_").expect("parse of S K y_");
-    ///
-    /// let alpha_1 = Term::alpha(&t,&t2);
-    /// let alpha_2 = Term::alpha(&t,&t3);
-    /// let alpha_3 = Term::alpha(&t2,&t3);
-    /// assert_ne!(alpha_1,alpha_2);
-    /// assert_eq!(alpha_2,alpha_3);
+    /// 
+    /// let vars = sig.variables();
+    /// let y = vars[0];
+    /// let z = vars[1];
+    /// let a = vars[2];
+    /// let b = vars[3];
+    /// assert_eq!(y.display(&sig), "y_".to_string());
+    /// assert_eq!(z.display(&sig), "z_".to_string());
+    /// assert_eq!(a.display(&sig), "a_".to_string());
+    /// assert_eq!(b.display(&sig), "b_".to_string());
```

Comment:
The variable names are informative enough that these four asserts aren't necessary. Can you get rid of them?

To make the example shorter, you can also write:

```rust
let y = vars[0];
let z = vars[1];
let a = vars[2];
let b = vars[3];
```
as
```rust
let (y, z, a, b) = (vars[0], vars[1], vars[2], vars[3]);
```

That technique might be useful in some of the other tests, too.